Authentication/Login/View/LoginController.swift

```diff
@@ -157,9 +149,9 @@ private extension LoginController {
         loginView.logInButton.rex_pressed.value = _viewModel.logInCocoaAction
         loginView.logInButton.rex_enabled.signal.observeNext { [unowned self] in self.loginView.logInButtonEnabled = $0 }
         
-        loginView.registerLabel.text = _viewModel.registerLabelText
-        loginView.registerButton.setTitle(_viewModel.registerButtonTitle, forState: .Normal)
-        loginView.registerButton.setAction { [unowned self] _ in self._transitionDelegate.loginControllerDidTapOnRegister(self) }
+        loginView.signupLabel.text = _viewModel.signupLabelText
+        loginView.signupButton.setTitle(_viewModel.signupButtonTitle, forState: .Normal)
```

Comment:
The only thing I don't like is to have localized string scattered through more than one class. And as the password visibility button's title need information from view model, I think I would leave them all in the view model.
Or maybe make a function in the view to get the localized string from a bool or directly change the title by itself, and make the controller notify when the password visibility property changes.
I'm okey with the second one